src/main/java/mate/academy/spring/model/User.java

```diff
@@ -16,6 +19,8 @@ public class User {
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany
+    private Set<Role> roles;
```

Comment:
This is the right approach, but I would add the following construction for better normalization of the database, namely the names of the columns in the database:
 @ManyToMany
    @JoinTable(name = "users_roles",
            joinColumns = @JoinColumn(name = "user_id"),
            inverseJoinColumns = @JoinColumn(name = "role_id"))
    private Set<Role> roles;

Reply:
@V-Solianyk 
It is good point in general, but for this task it is not very critical.